package.json

```diff
@@ -34,16 +34,14 @@
     "babel-jest": "^24.1.0",
     "babel-loader": "^8.0.5",
     "css-loader": "^2.1.0",
-    "enzyme": "^3.8.0",
-    "enzyme-adapter-react-16": "^1.9.1",
     "express": "^4.16.4",
     "file-loader": "^3.0.1",
     "google-fonts-webpack-plugin": "^0.4.4",
     "html-webpack-plugin": "^3.2.0",
     "materialize-css": "^1.0.0-rc.2",
     "node-sass": "^4.11.0",
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0",
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0",
```

Comment:
@ebenezerdon Is there any reason why you changed the versions of the react and react-dom packages?

Reply:
The caret(^) symbol before the packages updates packages to the most recent major version when we run `npm install`

Reply:
Ok. Thanks for clarifying. Though I was thinking that the version number would increase to say `16..9.0` as against `16.7.0` is there a reason for that please?

Reply:
I suspect `16.8.0` was the latest major version the last time he ran `npm install`.